pages/common/nginx.md

```diff
@@ -8,4 +8,4 @@
 
 - Start server with custom config and custom relative path base directory
 
-`nginx -c {{path to config}} -p {{relative path base dir}}`
+`nginx -c {{config_file}} -p {{relative/path/to/base/dir}}`
```

Comment:
Actually the second parameter denotes to which base dir relative paths in the config are to be resolved. This wording changes the meaning.